webpack.config.js

```diff
@@ -3,11 +3,30 @@ const glob = require('glob');
 
 const { InjectManifest } = require('workbox-webpack-plugin');
 const { SubresourceIntegrityPlugin } = require('webpack-subresource-integrity');
-const LicenseWebpackPlugin = require('license-webpack-plugin').LicenseWebpackPlugin;
+const LicensePlugin = require('webpack-license-plugin')
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 
+function manualLicenseInfo(package) {
+  return {
+    'jsondiffpatch': {
+      'spdx': 'MIT',
+      'detail': 'SEE LICENSE IN https://github.com/benjamine/jsondiffpatch/blob/a8cde4c666a8a25d09d8f216c7f19397f2e1b569/package.json#L81'
+    },
+    'slipjs': {
+      'spdx': 'BSD-2-Clause',
+      'detail': 'SEE LICENSE IN https://github.com/kornelski/slip/blob/91c24e460dbadb9e0dc40daf93fd01928bfac94d/package.json#L18'
+    }
+  }[package] || {'spdx': '', 'detail': package};
```

Comment:
Maybe remove the fallback case here; or, better, add an instructive error message if we fail to find licensing information for a package.